### PR TITLE
add nurse-component

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: configuration/encountertypes/encounterTypes.csv [hunk 1]
```diff
@@ -60,3 +60,4 @@ ${encounterType.RADIOLOGY_REPORT.uuid},,Rapport de radiologie,Radiology Report -
 ${encounterType.RADIOLOGY_STUDY.uuid},,Examen de radiologie,Radiology Study - represents performance of a radiology study on a patient by a radiology technician
 ${encounterType.ATTACHMENT_UPLOAD.uuid},TRUE,Attachment Upload,Encounters used to record uploads of attachments.
 ${encounterType.REHAB_EVAL.uuid},,Rehab Evaluation,A rehabilitation encounter with an initial or ongoing evaluation
+${encounterType.NURSE_CONSULT.uuid},,Nurse Consult,A consult with a nurse
```
Comment (mogoodrich) [inline configuration/encountertypes/encounterTypes.csv:63]: add a new "nurse consult" encounter type to be installed by Iniz during startup

File: configuration/messageproperties/messages_en.properties [hunk 2]
```diff
@@ -1921,6 +1921,7 @@ ui.i18n.EncounterType.name.b6631959-2105-49dd-b154-e1249e0fbcd7=Exit from Inpati
 ui.i18n.EncounterType.name.436cfe33-6b81-40ef-a455-f134a9f7e580=Transfer
 ui.i18n.EncounterType.name.92fd09b4-5335-4f7e-9f63-b2a663fd09a6=Consultation
 ui.i18n.EncounterType.name.59e55ca8-f3de-4d1e-b54c-9b880405b853=Initial Consultation
+ui.i18n.EncounterType.name.e920911f-dee9-4b18-9a58-50e685806ea0=Nurse Consultation
```
Comment (mogoodrich) [inline configuration/messageproperties/messages_en.properties:1924]: Provide a localizable display name for this new encounter type

File: configuration/pih/scripts/visit/encounterTypeConfig.js [hunk 3]
```diff
@@ -633,6 +633,17 @@ angular.module("encounterTypeConfig", [])
             showOnVisitList: true
         };
 
+      // nurse consultation
+      encounterTypes["e920911f-dee9-4b18-9a58-50e685806ea0"] = {
```
Comment (mogoodrich) [inline configuration/pih/scripts/visit/encounterTypeConfig.js:637]: configure the display of the nurse consultation with in the "Visit Note" view

File: constants.yml [hunk 4]
```diff
@@ -200,6 +200,8 @@ encounterType:
     uuid: "5021b1a1-e7f6-44b4-ba02-da2f2bcf8718"
   REHAB_EVAL:
     uuid: "e91a4139-e0e7-447f-a5dd-c4f3b92d27c9"
+  NURSE_CONSULT:
+    uuid: "e920911f-dee9-4b18-9a58-50e685806ea0"
```
Comment (mogoodrich) [inline constants.yml:204]: extract the uuid for the Nurse Consult out into a variable so that it will be compiled and availalbe in code

**Review by MiguelAHPpih — APPROVED** [review]
Looks good

**Pull request merged**